xdebug.ini

```diff
@@ -1,5 +1,5 @@
 variables_order = "EGPCS"
-xdebug.remote_host = 10.0.75.1
+xdebug.remote_host = ${LANDO_HOST_IP}

```

Comment:
I'd rather this be 'host.docker.internal' but this is fine.  See https://docs.docker.com/docker-for-mac/networking/#use-cases-and-workarounds

Reply:
I didn't know that. Thanks for the info. I found in Lando document "LANDO_HOST_IP=host.docker.internal".